metaseq/tasks/streaming_language_modeling.py

```diff
@@ -511,7 +519,14 @@ def get_previous_shard_str(self, epoch, split):
         prev_shard_str = shards[shard_idx - 1] if shard_idx > 0 else None
         return prev_shard_str
 
-    def load_dataset(self, split: str, epoch=1, combine=False, **kwargs):
+    def load_dataset(
+        self, split: str, epoch=1, combine=False, num_shards=-1, shard_id=-1, **kwargs
+    ):
+        is_sharded = True
+        if num_shards < 0 or shard_id < 0:
+            num_shards = 1
```

Comment:
assert num_shards < 0 && shard_id < 0  ? :P

Reply:
some parts of the code pass in None now (for example validation).